<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particulas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Particulas</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="3" column="9">
            <widget class="QSpinBox" name="B_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="1" column="4" colspan="3">
            <widget class="QSpinBox" name="origenX_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="1" column="8" colspan="2">
            <widget class="QSpinBox" name="origenY_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="10">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>MOSTRAR</string>
             </property>
            </widget>
           </item>
           <item row="2" column="8" colspan="2">
            <widget class="QSpinBox" name="destinoY_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="10">
            <widget class="QPlainTextEdit" name="salida"/>
           </item>
           <item row="1" column="7">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="10">
            <widget class="QPushButton" name="Final_pushButton">
             <property name="text">
              <string>AGREGAR FINAL</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>R</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>ORIGEN</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3" colspan="4">
            <widget class="QSpinBox" name="velocidad_spinBox">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="10">
            <widget class="QPushButton" name="distanciaOrdenar_pushButton">
             <property name="text">
              <string>ORDENAR POR DISTANCIA</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0" colspan="10">
            <widget class="QPushButton" name="idOrdenar_pushButton">
             <property name="text">
              <string>ORDENAR POR ID</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>VELOCIDAD</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="10">
            <widget class="QPushButton" name="Inicio_pushButton">
             <property name="text">
              <string>AGREGAR INICIO</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="6">
            <widget class="QLineEdit" name="ID_lineEdit"/>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>COLOR</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>DESTINO</string>
             </property>
            </widget>
           </item>
           <item row="2" column="7">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3" colspan="2">
            <widget class="QSpinBox" name="R_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="10">
            <widget class="QPushButton" name="grafos_pushButton">
             <property name="text">
              <string>MOSTRAR GRAFOS</string>
             </property>
            </widget>
           </item>
           <item row="3" column="6" colspan="2">
            <widget class="QSpinBox" name="G_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="3" column="8">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="10">
            <widget class="QPushButton" name="velocidadOrdenar_pushButton">
             <property name="text">
              <string>ORDENAR POR VELOCIDAD</string>
             </property>
            </widget>
           </item>
           <item row="3" column="5">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>G</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4" colspan="3">
            <widget class="QSpinBox" name="destinoX_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="3">
         <widget class="QTableWidget" name="tabla"/>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="buscar_lineEdit">
          <property name="placeholderText">
           <string>ID Particula</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buscar_pushButton">
          <property name="text">
           <string>BUSCAR</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="mostrar_tabla_pushButton">
          <property name="text">
           <string>MOSTRAR</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Particula</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="dibujar">
          <property name="text">
           <string>DIBUJAR</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="limpiar">
          <property name="text">
           <string>LIMPIAR</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>469</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <widget class="QMenu" name="menuAlgoritmos">
    <property name="title">
     <string>Algoritmos</string>
    </property>
    <addaction name="actionRecorrido_Amplitud"/>
    <addaction name="actionRecorrido_Profundidad"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAlgoritmos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRecorrido_Amplitud">
   <property name="text">
    <string>Recorrido Amplitud</string>
   </property>
  </action>
  <action name="actionRecorrido_Profundidad">
   <property name="text">
    <string>Recorrido Profundidad</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
